CSS-files-e-com/login.css

```diff
@@ -0,0 +1,61 @@
+.login-wrapper{
+    display: flex;
+    margin: 15rem auto;
+    flex-direction: column;
+    gap: 1rem;
+    align-items: center;
+}
+.submit-form{
+    display: flex;
+    flex-direction: column;
+    gap: 1rem;
+    width: 100%;
+    position: relative;  
+    max-width: 40rem;
+}
+.input-primary{
+    padding: .6rem;
+    width: 100%;
+    border-radius: .4rem;
+    border: 1px solid var(--body-bg-color);
+    outline: none;
+    line-height: 1rem;
+    
+}
+.input-primary:focus{
+    box-shadow: 0 0 0 1px var(--danger-message-color);   
+}
+.eye-btn{
+    position: absolute;
+    top: 4.1rem;
+    right: .5rem;
+    border: none;
+}
+.btn-login{
+    padding: .4rem;
+    border-radius: .4rem;
+    /* outline: none; */
```

Comment:
You can remove this comment if not needed

Reply:
have kept it for future reference